<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subwindow</class>
 <widget class="QDialog" name="subwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="SliderBar" name="widgetNavLeft" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>150</width>
     <height>621</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widgetMain" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>731</width>
     <height>631</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="passengerGroupBox">
       <property name="title">
        <string>乘客</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="currentAccount">
          <property name="title">
           <string>当前账号</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="nameLable">
             <property name="text">
              <string>姓名</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="nameShow">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="idLable">
             <property name="text">
              <string>身份证号</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="idShow">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="buyPage">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="1">
             <widget class="QGroupBox" name="buyGroupBox">
              <property name="title">
               <string>购票</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="startStationLable">
                 <property name="text">
                  <string>始发站</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLabel" name="seatLabel">
                 <property name="text">
                  <string>席别</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QComboBox" name="startProvince">
                 <item>
                  <property name="text">
                   <string>北京</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>河北</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="2" colspan="2">
                <widget class="QComboBox" name="startCity">
                 <item>
                  <property name="text">
                   <string>北京</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QComboBox" name="seatComboBox">
                 <item>
                  <property name="text">
                   <string>硬座</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>卧铺</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>站票</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>动车</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="endLable">
                 <property name="text">
                  <string>终点站</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="4">
                <widget class="QLabel" name="numberLable">
                 <property name="text">
                  <string>购票张数</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QComboBox" name="endProvince">
                 <item>
                  <property name="text">
                   <string>北京</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>河北</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="2" colspan="2">
                <widget class="QComboBox" name="endCity">
                 <item>
                  <property name="text">
                   <string>北京</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="4">
                <widget class="QSpinBox" name="numberSpinBox"/>
               </item>
               <item row="4" column="0" colspan="5">
                <widget class="QCalendarWidget" name="calendarWidget"/>
               </item>
               <item row="5" column="0">
                <widget class="QPushButton" name="priceShowButton">
                 <property name="text">
                  <string>显示价格</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="4">
                <widget class="QPushButton" name="purchaseButton">
                 <property name="text">
                  <string>购买</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1" colspan="3">
                <widget class="QLabel" name="priceShowLable">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="lineWidth">
                  <number>1</number>
                 </property>
                 <property name="midLineWidth">
                  <number>0</number>
                 </property>
                 <property name="text">
                  <string>0￥</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="refundPage">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QGroupBox" name="refundGroupBox">
              <property name="title">
               <string>退票</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="refundLabel">
                 <property name="text">
                  <string>输入序号</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="refundLineEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QPushButton" name="refundButton">
                 <property name="text">
                  <string>退票</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="endorsePage">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>改签</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0" colspan="2">
                <widget class="QCalendarWidget" name="calendarWidget_2"/>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="submitPushButton">
                 <property name="text">
                  <string>提交</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="CancelPushButton">
                 <property name="text">
                  <string>放弃</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetTop" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="labTitle">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="SliderBar" name="widgetNavTop" native="true"/>
    </item>
    <item>
     <spacer name="horizontalSpacer2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widgetMenu" native="true">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnMenu_Min">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>最小化</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnMenu_Close">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>关闭</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnMenu_Max">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderBar</class>
   <extends>QWidget</extends>
   <header location="global">sliderbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
